force_bdss/io/tests/test_workflow_reader.py

```diff
@@ -31,61 +36,103 @@ def setUp(self):
         self.wfreader = WorkflowReader(self.registry)
         self.working_data = fixtures.get("test_workflow_reader.json")
 
-    def test_initialization(self):
-        self.assertEqual(self.wfreader.factory_registry, self.registry)
-
-        workflow = self.wfreader.read(self.working_data)
-
-        self.assertIsInstance(workflow, Workflow)
-
     def test_load_data(self):
         data = self.wfreader.load_data(self.working_data)
         control_dict = {
```

Comment:
Is it possible to add an additional dictionary to test continuing support for version 1 workflow JSONs?

Reply:
Sorry, it was in my "to push" stack. Should be there now.

Reply:
Tests added.